server/proxy/server.js

```diff
@@ -143,6 +143,22 @@ app.get('/playCard', (req, res) => {
     .catch((err) =>  res.send(err));
 });
 
+app.get('/voteCard', (req, res) => {
```

Comment:
Struggling to understand what this is doing? Is there a danger of the user being able to pass in their own url here that theyre hosting? or would that even matter?

Reply:
I'm not really sure what you mean... this proxy server is ran only during testing mode though

Reply:
ahhh! got you, nvm!